nautobot/dcim/models/devices.py

```diff
@@ -1293,26 +1298,51 @@ class Controller(PrimaryModel, ConfigContextModel):
         null=True,
     )
 
+    class Meta:
+        ordering = ("name",)
+
+    def __str__(self):
+        return self.display or super().__str__()
```

Comment:
IIRC the display attr is a property that is generated using the `__str__`. You should not create the display property